website/source/docs/configuration/storage/postgresql.html.md

```diff
@@ -96,6 +96,11 @@ LANGUAGE plpgsql;
   which to write Vault data. This table must already exist (Vault will not
   attempt to create it).
 
+- `max_idle_connections` `(int)` - Default not set. Sets the maximum number of 
+  connections in the idle connection pool. See
+  [golang docs on SetMaxIdleConns][golang_SetMaxIdleConns] for more information. 
+  Requires 1.2 or later.
```

Comment:
Since you tagged this as part of the 1.2 milestone I updated the docs to say this param requires 1.2 or later.